packages/tailwind-config/tailwind.config.mjs

```diff
@@ -16,6 +16,7 @@ export default {
     colors: {
       white: '#FFFFFF',
       black: '#000000',
+      secondary: "#323D37",
```

Comment:
let's use the color that is already defined: `neutralGreen-700`

Reply:
i remove this variable